### DISCUSSION TITLE
Do you have any idea with two distinct containers for docker

### DISCUSSION THREAD

**Discussion by roukmoute** [phpro/grumphp#971] [Q&A] [answered]
Hey!

I've got two containers, one for PHP and one for NPM.

I have currently this:

```yml
    EXEC_GRUMPHP_COMMAND: [ 'docker-compose', 'exec', '-T', 'php', 'php', '-d', 'memory_limit=-1' ]
```

Do you have any idea how can I run my NPM on hooks?

Thanks in advance

**Comment by veewee** [comment, marked as answer]
It is not possible to run GrumPHP on multiple containers.
You can do so by creating a separate container that contains both php and node.

**Reply by roukmoute** [reply]
that's what I was afraid of but I understand, thanks for your speed!
